### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by schemen**

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc [hunk 1]
```diff
@@ -166,6 +166,13 @@ For the following steps, change into a clean directory (for example a directory
 These are the only steps which aren't idempotent and have to be completed uninterrupted in one go.
 If you have to recreate the install config or any of the generated manifests you need to rerun all of the subsequent steps.
 
+[NOTE]
+--
+You can add options to the `install-config.yaml` as per upstream configuration. Have a look at the https://docs.openshift.com/container-platform/4.5/installing/installing_bare_metal/installing-bare-metal.html#installation-bare-metal-config-yaml_installing-bare-metal[config example].
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc:171]:
```suggestion
You can add options to the `install-config.yaml` as per upstream configuration.
Have a look at the https://docs.openshift.com/container-platform/4.5/installing/installing_bare_metal/installing-bare-metal.html#installation-bare-metal-config-yaml_installing-bare-metal[config example].
```

New line per sentences as per https://syn.tools/syn/references/style-guide.html#_asciidoc_style

File: docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc [hunk 2]
```diff
@@ -246,23 +253,30 @@ CA_CERT=$(jq -r '.ignition.security.tls.certificateAuthorities[0].source' \
 
 pushd "inventory/classes/${TENANT_ID}/"
 
-yq w -i "${CLUSTER_ID}.yml" \
-  "applications[+]" "openshift4-cloudscale"
+yq eval -i ".classes +=  \"${TENANT_ID}.openshift4\"" \
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc:256]: Please update the requirments within that document. State the minimal `yq` version required.

**Review by corvus-ch — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc (hunk 1 above)
Comment (ccremer) [reply docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc:171]: I'm having a hard time understanding "as per upstream configuration"...
Doesn't sound like proper English to me. What did you mean? "Similar to...", something else?
Prolly best to reword this sentence

Comment (schemen) [reply docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc]: I have updated the sentence, please let me know if this is more understandable.

**Review by corvus-ch — APPROVED** [review]

File: docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc [hunk 3]
```diff
@@ -166,6 +166,14 @@ For the following steps, change into a clean directory (for example a directory
 These are the only steps which aren't idempotent and have to be completed uninterrupted in one go.
 If you have to recreate the install config or any of the generated manifests you need to rerun all of the subsequent steps.
 
+[NOTE]
+--
+You can add more options to the `install-config.yaml` file. 
+Have a look at the https://docs.openshift.com/container-platform/4.5/installing/installing_bare_metal/installing-bare-metal.html#installation-bare-metal-config-yaml_installing-bare-metal[config example] for more information.
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc:172]: Should we go for 4.6 already?

File: docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc [hunk 4]
```diff
@@ -246,23 +254,30 @@ CA_CERT=$(jq -r '.ignition.security.tls.certificateAuthorities[0].source' \
...
 
-yq w -i "${CLUSTER_ID}.yml" \
-  parameters.openshift4_cloudscale.variables.base_domain -- "${BASE_DOMAIN}"
-yq w -i "${CLUSTER_ID}.yml" \
-  parameters.openshift4_cloudscale.variables.ignition_ca -- "${CA_CERT}"
 
-yq w -i "${CLUSTER_ID}.yml" \
-  parameters.vshnLdap.serviceId -- ${LDAP_ID}
+# Have a look at the file ${CLUSTER_ID}.yml, you can check if you want to add any specific instructions other than default values
+# or add component configuration.
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc:280]:
```suggestion
# Have a look at the file ${CLUSTER_ID}.yml.
# Override any default parameters or add more component configuration.
```

File: docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc [hunk 5]
```diff
@@ -33,7 +33,7 @@ It's currently very specific to VSHN and needs further changes to be more generi
 * `docker`
 * `mc` https://docs.min.io/docs/minio-client-quickstart-guide.html[Minio client] (aliased to `mc` if necessary)
 * `jq`
-* `yq` https://mikefarah.gitbook.io/yq[yq YAML processor]
+* `yq` https://mikefarah.gitbook.io/yq[yq YAML processor] (Minimum version 4)
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc:36]:
```suggestion
* `yq` https://mikefarah.gitbook.io/yq[yq YAML processor] (version 4 or higher)
```

File: docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc (hunk 3 above)
Comment (schemen) [reply docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc:172]: You have to tell me, current setups are still done in 4.5 as far as I am aware.

Comment (corvus-ch) [reply docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc:172]: I have had the same thought but No, not yet. I will change that as part of my current task.

**Review by ccremer — APPROVED** [review]
Be sure to squash/clean the commit history before merging. thx!

**Pull request merged**